app/presenters/claim/base_claim_presenter.rb

```diff
@@ -83,12 +83,10 @@ def case_type_name
   end
 
   def defendant_names
-    defendant_names = claim.defendants.sort_by(&:id).map(&:name)
-
     h.capture do
-      defendant_names.each do |name|
+      defendant_names_list.each_with_index do |name, idx|
```

Comment:
Is this something that would be useful elsewhere? Something like this;

```ruby
def defendant_names
  defendant_names = claim.defendants.sort_by(&:id).map(&:name)

  capture_join(defendant_names, [', ', h.tag.br])
end

def capture_join(elements, links)
  h.capture do
    elements.each_with_index do |element, idx|
      h.concat(name)
      break if idx == element.size - 1
      links.each { |link| h.concat(link) }
    end
  end
end
```

I haven't tried this but I hope it gives the general idea. It feels like the sort of thing that could occur in a number of different places.

Reply:
@jrmhaig no to my knowledge so would be overkill for now.  context of this is to fix a cop small cop violation. if there are no blockers can you approve?!

Reply:
:+1: OK, no other blockers.